ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -799,20 +804,38 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
                  contentRight={specificPopulations.map(sp => sp.ubrDescription)}/>
             </div>
           }
-          <div style={{display: 'flex', flexDirection: 'row',
-            paddingBottom: '14.4px', paddingTop: '0.3rem'}}>
-            <CheckBox style={{height: '.66667rem', marginRight: '.31667rem', marginTop: '0.3rem'}}
-              onChange={v => this.setState(
-                fp.set(['workspace', 'researchPurpose', 'reviewRequested' ], v))}
-              checked={this.state.workspace.researchPurpose.reviewRequested}/>
+          <div style={{display: 'flex', flexDirection: 'row', paddingTop: '0.3rem'}}>
```

Comment:
optional,
there's a `FlexRow` element you can use instead of `<div style={{display: 'flex', flexDirection: 'row' ...`
like:
`<FlexRow style={{paddingTop: '0.3rem'}}>`

Reply:
Sounds like a good target for a mass-refactoring!  I'll do that in a separate PR